_drafts/2017-04-18-tabulizer.Rmd

```diff
@@ -0,0 +1,205 @@
+---
+name: tabulizer
```

Comment:
Thank you Thomas for this great post. Love the narrative and I have only minor comments below. @sckott might have additional comments, but looks good to go from my perspective.